android-base/src/main/java/io/github/droidkaigi/confsched2020/MainActivity.kt

```diff
@@ -92,8 +94,21 @@ class MainActivity : DaggerAppCompatActivity() {
         setupNavigation()
         setupStatusBarColors()
 
-        binding.drawerLayout.doOnApplyWindowInsets { _, insets, _ ->
+        binding.drawerLayout.doOnApplyWindowInsets { _, insets, initialState ->
             binding.drawerLayout.setChildInsetsWorkAround(insets)
+            binding.contentContainer.updatePadding(
```

Comment:
Probably you should add these callbacks 🙇 🙇 
Because of initialState is that `view.doOnApplyWindowInsets` view margin or paddings

```
binding. contentContainer.doOnApplyWindowInsets { _, insets, initialState ->
```

```
binding. navView.doOnApplyWindowInsets { _, insets, initialState ->
```